Makefile

```diff
@@ -78,3 +78,12 @@ curl_local curl_prod:
 		"N":["C7", "H6", "H7", "H9", "CJ", "SA", "S8", "SQ", "D5", "S5", "HK", "C8", "HA"],    \
 		"E":["H2", "H5", "CQ", "D9", "H4", "ST", "HQ", "SJ", "HJ", "DQ", "H3", "C9", "CK"] }}' \
 	${CURL_URL}
+
+logfile := logs/src.api.$(shell date +%FT%T).log
+pidfile := logs/server.PID
+
+start_local_server: libdds-build stop_local_server
+	python3 -m src.api > ${logfile} 2>&1 & echo $$! > ${pidfile}
+
```

Comment:
Should there be also dependency from local_curl to start_local_server?

Dependencies can be added by declaring target with the dependency without any commands.

```
local_curl: start_local_server
```

Reply:
I don't think this is a good idea. I don't want to start a background process behind the user's back. We'd also need to insert a delay somewhere since the curl will fill while the server is starting up.